### DISCUSSION TITLE
Quick Reply

### DISCUSSION THREAD

**Discussion by microbearsmb** [protz/thunderbird-conversations#1647] [Q&A]
Is there a timeline for when Quick Reply will be implemented in Thunderbird 78+? It is a feature I would love to see, so I hope there is still active development of it.

**Comment by Standard8** [comment]
This has been slowly progressing - see the comments in #1361 and #1513.

At the moment, there's very simple message sending (no from/to editing) that's available in a separate compose window, and isn't quite fully integrated into the message pane yet. It also needs a lot of styling re-implementing.

I'm not quite sure when it'll be ready for a minimal version for release, unfortunately I haven't been able to spend as much time as I'd like, though I'm thinking it probably isn't too far off now.

**Reply by microbearsmb** [reply]
Alright, thanks!
